Dockerfile

```diff
@@ -1,32 +1,38 @@
-# SPDX-License-Identifier: Apache-2.0
-# Copyright Contributors to the Egeria project
+# Stage 1 - Build
+FROM node:16.13.0 AS builder
 
-FROM public.ecr.aws/nginx/nginx:latest
+WORKDIR /app
+COPY . /app
+RUN npm install
+RUN npm run build:docker
```

Comment:
How does this change the docker build? any impact on supporting multiple architectures?